# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-08-26 13:59-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: src/services/authentication.py:34 src/services/authentication.py:36
#: src/services/authentication.py:41
msgid "invalid access token"
msgstr ""

#: src/services/authentication.py:64 src/services/authentication.py:70
msgid "invalid username or password"
msgstr ""

#: src/services/authentication.py:67
msgid "this user has been disabled"
msgstr ""

#: src/routers/role.py:38 src/routers/role.py:54 src/routers/role.py:86
msgid "role not found"
msgstr ""

#: src/routers/role.py:70
msgid "bad role name"
msgstr ""

#: src/routers/users.py:26 src/routers/users.py:56 src/routers/users.py:79
#: src/routers/users.py:103
msgid "current user not found"
msgstr ""

#: src/routers/users.py:64 src/routers/users.py:88 src/routers/users.py:108
msgid "user not found"
msgstr ""

#: src/routers/users.py:83
msgid "not allowed delete current user"
msgstr ""

#: src/routers/users.py:113
msgid "username already exists"
msgstr ""

#: src/routers/permission.py:38 src/routers/permission.py:54
#: src/routers/permission.py:88
msgid "permission not found"
msgstr ""

#: src/routers/permission.py:70
msgid "bad permission name"
msgstr ""

#: src/routers/permission.py:72
msgid "bad permission uri"
msgstr ""

#: src/routers/register.py:33
msgid "user with this username already exists"
msgstr ""

#: src/routers/register.py:36
msgid "user with this email already exists"
msgstr ""
